bot/define_command.go

```diff
@@ -0,0 +1,58 @@
+package bot
+
+import (
+	"fmt"
+	"io"
+	"net/url"
+	"strings"
+
+	"github.com/urfave/cli"
+	"github.com/zpatrick/rclient"
+)
+
+// DatamuseAPIEndpoint for the datamuse api
+const DatamuseAPIEndpoint = "https://api.datamuse.com"
+
+// DatamuseResponse is the response type for a Tenor API search
+type DatamuseResponse []struct {
+	Definitions []string `json:"defs"`
+	Word        string   `json:"word"`
+}
+
+// NewDefineCommand returns a cli.Command that manages !gif
+func NewDefineCommand(endpoint string, w io.Writer) cli.Command {
+	client := rclient.NewRestClient(endpoint)
+	return cli.Command{
+		Name:        "!define",
+		Usage:       "display a definition for query",
+		Description: "A list of definitions is return with a preface of n for nouns or v is for verbs",
+		ArgsUsage:   "args...",
+		Action: func(c *cli.Context) error {
+			args := c.Args()
```

Comment:
ditto - shouldn't this just be like `word := c.Args().Get(0)` ? 

FWIW users can do `!define "ice cream"` for a word w/ spaces

Reply:
I think we should allow spaces, since the api does.

Reply:
It can take a word or phrase